### PR TITLE
903604-UG document for  .NET-MAUI Kanban control

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description** 

Updated review changes for Kanban control UG document.

### CODE REVIEW

File: MAUI/Kanban-Board/Column.md [hunk 1]
```diff
@@ -72,26 +71,6 @@ kanban.ColumnMappingPath = "ID";
 
 {% endtabs %}
 
-### Multiple category for a column
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Column.md:75]: @EzhilarasanElangovan31 update the content for single column -> Category for a column and update content and code snippet

File: MAUI/Kanban-Board/Workflows.md [hunk 2]
```diff
@@ -36,5 +36,4 @@ kanban.Workflows = workflows;
 
 {% endhighlight %}
 
-
-In the below output, you can see the card which was picked from Open state is not allowed to drop on “Code Review” and "Done" state, because we have defined to move the card from Open to “In Progress” state only and not to any other states.
\ No newline at end of file
+The card picked from the "Open" state is not allowed to be dropped in the "Code Review" or "Done" states because we have defined that the card can only move from the "Open" state to the "In Progress" state and not to any other states.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Workflows.md:39]: @EzhilarasanElangovan31 update as -> By following the code snippet, you will notice that the card picked....

File: maui-toc.html [hunk 3]
```diff
@@ -723,6 +710,19 @@
...
+    <li>
+        SfKanban
+        <ul>
+            <li><a href="/maui/kanban-board/overview">Overview</a></li>
+            <li><a href="/maui/kanban-board/Getting-Started">Getting Started</a></li>
+            <li><a href="/maui/kanban-board/migration">Migrate from Xamarin.Forms</a></li>
+            <li><a href="/maui/kanban-board/column">Column</a></li>
+            <li><a href="/maui/kanban-board/workflows">Workflows</a></li>
+            <li><a href="/maui/kanban-board/cards">Cards</a></li>
+            <li><a href="/maui/kanban-board/events">Events</a></li>
```
Comment (VimalaThirumalaiKumarSF3739) [inline maui-toc.html:722]: @EzhilarasanElangovan31 change the file order, update Localization before Events

File: MAUI/Kanban-Board/Column.md (hunk 1 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Column.md:75]: Updated the content

File: MAUI/Kanban-Board/Workflows.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Workflows.md]: Updated

File: maui-toc.html (hunk 3 above)
Comment (EzhilarasanElangovan31) [reply maui-toc.html]: changed the order

**Comment by VimalaThirumalaiKumarSF3739** [comment]
Fine

**Review by SyncfusionKarthikeyan — APPROVED** [review]
This is fine

**Pull request merged**